packages/salesforcedx-vscode-core/src/notifications/progressNotification.ts

```diff
@@ -35,6 +36,19 @@ export class ProgressNotification {
           execution.processErrorSubject.subscribe(data => {
             return resolve();
           });
+
+          if (progressReporter) {
```

Comment:
This enables reporting incremental progress updates and (more importantly for the new command) to be able to signify task completion while the child process itself continues to run.